docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -117,6 +117,7 @@ Start a new line at the end of each sentence, and split sentences themselves at
 === Abstracts (preamble)
 
 For optimum findability, add a short description using user-oriented keywords to summarize the document's purpose, task, and intent in no more than three sentences.
+Good abstracts also mention the benefits of following this content and avoid filler words, such as: "This document..", "This tutorial..", and "The following.."
```

Comment:
This is a good addition:) 
I have 3 suggestions: 
* Please consider splitting this sentence into 2 shorter ones.
* If you choose to follow the first recommendation, I suggest you swap the order of the 2nd and 3rd sentences (see example at the end of my comment.)
* The use of the phrase _this content_ is a bit ambiguous, because it might not be apparent what _this_ is referring to. You might want to rephrase your sentence a little. For example: 

_Avoid self referential expressions and filler words, such as "This document..", "This tutorial..", and "The following.."
You should also mention what benefits users gain from reading the content that you are introducing._


Otherwise, LGTM! Nice work!